ego4d/internal/validate.py

```diff
@@ -593,6 +604,11 @@ def validate_university_files(  # noqa :C901
         write.writerow(fields)
         for e in error_message:
             write.writerow([e.uid, e.errorType, e.description])
+    with open("error_summary", "w") as f:
```

Comment:
We should use variables to describe the output file paths, as configuration options to the CLI tool, e.g. `summary_output_path` and `error_output_path`

and when running pass in e.g. `--summary_output_path error_summary`